### ISSUE TITLE
Recovery Rate and Mortality Rate wrong calculation

### ISSUE THREAD

**Issue by saurabhsuniljain** [covid19india/covid19india.github.io#2218]
**Describe the bug**
Recovery Rate and Mortality Rate are calculated as below on the current website:
Recovery Rate = Recovered / Confirmed * 100 
Mortality Rate = Deceased / Confirmed * 100

whereas it should be calculated as 
Recovery Rate = Recovered / [Recovered + Deceased] * 100 
Mortality Rate = Deceased / [Recovered + Deceased] * 100

On the world meter website, it is calculated the correct way. You can check https://www.worldometers.info/coronavirus/country/india/
For `Closed Cases` snippet at the top and `Outcome of Cases (Recovery or Death) in India` graph

**To Reproduce**
Present on the live website.

**Expected behavior**
Recovery Rate and Mortality Rate should be calculated using the below formulas
Recovery Rate = Recovered / [Recovered + Deceased] * 100 
Mortality Rate = Deceased / [Recovered + Deceased] * 100

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by sudevschiz** [comment]
@bee-rickey This needs more discussion. Should we change the wording and the calculation so that we show Case fatality rate **for cases where an outcome is known**?

**Comment by vhbelvadi** [comment]
@saurabhsuniljain – I think using confirmed cases is fairer because it errs on the side of underestimating the recovery rate rather than overestimating it. Using the (recovered + deceased) sum alone would potentially overestimate the rate until the status of unknown cases becomes clear, causing a rather confusing dip in the recovery rate from time to time.

Edit: Or we need to reword it to something like 'Instantaneous recovery rate'. Either way, in my opinion the approach World Meter uses isn't necessarily more 'correct'.

**Comment by sudevschiz** [comment]
Let's maintain status-quo and continue to use the overall confirmed cases to show the statistics. We will try to be explicit about the calculation in a help pane.
